### DISCUSSION TITLE
Counter-Strike 2 / CS:GO

### DISCUSSION THREAD

**Discussion by mvaessen** [gameserverapp/Platform#25] [Game request]
https://developer.valvesoftware.com/wiki/Counter-Strike:_Global_Offensive_Dedicated_Servers

**Comment by borzaka** [comment]
Some helping material for adding CS:GO in RconConnect:
- https://github.com/xPaw/PHP-Source-Query

**Comment by nghialele** [comment]
Why is this popular game not available (yet)? :(

**Reply by mvaessen** [reply]
Hi, you should be able to get this working quite easily using the LinuxGSM blueprint helper.
I talked to Dan, from LinuxGSM, and he said he got one up and running using that helper. Should be fairly simple.

https://docs.gameserverapp.com/dashboard/blueprints/how-to/create_linuxgsm_blueprint

Planning to add a couple LinuxGSM servers soon

**Comment by mvaessen** [comment]
Shout out to Echo for adding the CS2 & CS:GO blueprint to the marketplace. <3

This means CS2 & CSGO is now available on the dashboard for install on Linux

CS2: https://dash.gameserverapp.com/marketplace/market/blueprint/480
CS:GO: https://dash.gameserverapp.com/marketplace/market/blueprint/489
